### PR TITLE
Fixed original dev portal link in Developer Portal

### PR DESCRIPTION
**Pull request by Kirishikesan**
Fixed the issue : [Original Dev Portal link was not working](https://github.com/wso2/api-manager/issues/150)

### CODE REVIEW

**Review by chanaka3d — APPROVED** [review]

File: tests/cypress/support/commands.js [hunk 1]
```diff
@@ -359,3 +359,103 @@ Cypress.Commands.add('logoutFromPublisher', () => {
...
+    cy.get('#left-menu-itemRuntimeConfigurations').should('exist');
+    cy.get('#left-menu-itemresources').should('exist');
+    cy.get('#left-menu-itemLocalScopes').should('exist');
+    cy.get('#left-menu-itemMonetization').should('exist');
+    cy.get('#itest-api-details-api-config-acc').click();
+    cy.get('#left-menu-itemTestConsole').should('exist');
+
+    //Check if the api is not deployable
+    cy.get('#left-menu-itemdeployments').click();
+    cy.get('[data-testid="third-party-api-deployment-dialog"]').contains('This API is marked as a third party API. The requests are not proxied through the gateway. Hence, deployments are not required.').should('exist');
```
Comment (PasanT9) [inline tests/cypress/support/commands.js:425]: Line length exceeds 120 chars. Please check other places as well.

File: tests/cypress/support/commands.js [hunk 2]
```diff
@@ -359,3 +359,103 @@ Cypress.Commands.add('logoutFromPublisher', () => {
...
+    cy.get('[data-testid="itest-original-devportal-link"]').should('exist');
+    cy.get('[data-testid="itest-no-tier-dialog"]').contains('No tiers are available for the API.').should('exist');
+
+    //Check if authorization header and value can be customized
+    cy.get('#left-menu-test').click();
+    cy.get('#advAuthHeader').should('exist');
+    cy.get('#advAuthHeaderValue').should('exist');
+
+})
+
```
Comment (PasanT9) [inline tests/cypress/support/commands.js:460]: Remove additional new line. Leaving a single line is enough.

Comment (Kirishikesan) [reply tests/cypress/support/commands.js:460]: This is fixed in [apim-apps#426](https://github.com/wso2/apim-apps/pull/426)

File: tests/cypress/support/commands.js (hunk 1 above)
Comment (Kirishikesan) [reply tests/cypress/support/commands.js:425]: This is fixed in [apim-apps#426](https://github.com/wso2/apim-apps/pull/426)

**Pull request merged**